libs9s/s9srpcclient.cpp

```diff
@@ -10304,15 +10314,62 @@ S9sRpcClient::composeBackupJob()
     if (!snapshotLocation.empty())
         jobData["snapshot_location"] = snapshotLocation;
 
-    if (options->hasCredentialIdOption())
-        jobData["credential_id"] = options->credentialId();
-
-    if (!s3bucket.empty())
-        jobData["s3_bucket"] = s3bucket;
-
     if (!s3region.empty())
         jobData["s3_region"] = s3region;
 
+    if (onlyCloud)
```

Comment:
Support for this option should be separated to another method. This one is way too long.